Scheduler scrollTo: add details for the 2nd parameter (#2760)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxScheduler/3 Methods/scrollTo(date_group_allDay).md

> A date and time to scroll to.

Reviewer: ```suggestion
A date and time to which to scroll.
```

##### param(group): Object | undefined

> If appointments are [grouped by resources](/Documentation/Guide/UI_Components/Scheduler/Resources/Group_Appointments_by_Resources/), specifies the appointment group id (optional).

Reviewer 2: ```suggestion
If appointments are [grouped by resources](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/), specifies a group ID (optional).
```
